### ISSUE TITLE
Firebase 7.0.2 is not available in UPM

### ISSUE THREAD

**Issue by uchar** [firebase/quickstart-unity#948]
It seems that the latest release of firebase is not available in upm .
https://upm-packages.dev/-/web/detail/com.google.firebase.analytics

Can you please update it?

**Comment by patm1987** [comment]
Hi @uchar ,

Thanks for reaching out to us! Unfortunately, we don't actively maintain anything on upm-packages.dev. You can find our official installation instructions [here](https://firebase.google.com/docs/unity/setup).

Let me know if that helps, I'm not super familiar with the upm-packages offering, so I could be misunderstanding the request.

--Patrick

**Comment by uchar** [comment]
@patm1987
upm-package helps in two scenario
1- it doesn't make project structure and folders messy
2- One of firebase file is larger than 100mbs(it's 104mb) and I hit some problems with github actions, because if a file is larger than 100mbs, it needs git lfs and then the project CiCD goes out of their usage quoata only because of 4mb

**Comment by patm1987** [comment]
I totally understand (I am a fan of the Unity Package Manager myself). The package in that specific repository (upm-packages.dev) wasn't uploaded by the team here and I couldn't quickly find instructions on uploading or maintaining packages for that repository.

My two best suggestions are:
* If you can figure out who uploaded that package, your quickest path to resolution would be to ping them.
* If you'd like to share instructions with us about the package submission process, I can turn this into a feature request.

And definitely let me know if I'm misunderstanding the intent of the question.

**Comment by uchar** [comment]
@patm1987 
I actually upload [one for myself ](https://openupm.com/packages/com.dopamine.firebase.analytics/)in https://openupm.com/ ( not upm-packages.dev)
It's pretty straight forward, And there is a nice document of the upload process here https://openupm.com/docs/.

**Comment by chkuang-g** [comment]
@uchar 

Sorry we cannot upload our libraries to a third-party repo.  
We also will not stop you from doing so.  Just please make it very clear that this is your personal copy. (Ex. your domain name `com.dopamine` is a good start)

I will close this issue for now

**Comment by CoderGamester** [comment]
I'm following this issue and would love to ask for an update on the UPM issue.
I see that was removed from Firebase setup instructions.
As mentioned by @uchar , keeping things in the UPM is so much better for all possible reasons.
I found and follow the link https://laptrinhx.com/managing-the-firebase-unity-plugin-the-easy-way-1600281350/ to configure in my project.

My question is, isn't this registry made by Google?

Name: Game Package Registry by Google
URL: https://unityregistry-pa.googleapis.com
Scope(s): com.google

I can see there the latest updates of the Firebase.
